manifest.json

```diff
@@ -13,6 +13,17 @@
             "workflow_type": "modular"
         },
         "modular_c3.7s": {
+            "generator": {
```

Comment:
Why this changed?

Reply:
Ah sorry, this snuck in when I was testing the change in the same setup that I saw the original issue in. will remove